src/SPTesting.hs

```diff
@@ -0,0 +1,35 @@
+module SPTesting where
```

Comment:
Can these functions be used for tests automation? For example, for unit tests.

Reply:
I've created new module based on this module for unit tests